https://issues.redhat.com/browse/ACM-13045 for 2.11 (#6742)
open-cluster-management/rhacm-docs: observability/customize_observability.adoc

Create custom rules for the observability installation by adding Prometheus link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/[recording rules] and link:https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[alerting rules] to the observability resource.

> To precalculate expensive expressions, use the recording rules abilities. The results are saved as a new set of time series. With the alerting rules, you can specify the alert conditions based on how you want to send an alert to an external service. Define custom rules with Prometheus to create alert conditions, and send notifications to an external messaging service. View the following examples to create a custom alert rule within the `observability-thanos-rule-custom-rules` config map:

Reviewer: What do you think about making it a bullet list of capabilities? Something like: 

```suggestion
To pre-calculate expensive expressions, use the recording rules abilities. The results are saved as a new set of time series. With the alerting rules, you can do the following:
- Specify the alert conditions based on how you want to send an alert to an external service. 
- Define custom rules with Prometheus to create alert conditions. 
- Send notifications to an external messaging service. 
- View the following examples to create a custom alert rule within the `observability-thanos-rule-custom-rules` config map:
```

Author: Thanks for the suggestion. Originally, I was letting the user know how to define custom rules to create alert conditions (that's with Prometheus). I should have not used the comma after "conditions"

> To precalculate expensive expressions, use the recording rules abilities. The results are saved as a new set of time series. With the alerting rules, you can specify the alert conditions based on how you want to send an alert to an external service. Define custom rules with Prometheus to create alert conditions, and send notifications to an external messaging service. View the following examples to create a custom alert rule within the `observability-thanos-rule-custom-rules` config map:

Author: Based on the style guide, precalculate should remain the same: https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes
```suggestion
To precalculate expensive expressions, use the recording rules abilities with Prometheus to create alert conditions and send notifications based on how you want to send an alert to an external service. The results are saved as a new set of time series. View the following examples to create a custom alert rule within the `observability-thanos-rule-custom-rules` config map:
```

Define custom rules with Prometheus to create alert conditions, and send notifications to an external messaging service. View the following examples of custom rules:

> - Create a custom alert rule. Create a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace. You must name the key, `custom_rules.yaml`, as shown in the following example. You can create multiple rules in the configuration. 

Reviewer: Since there are several actions here, what do you think about breaking it up into multiple steps? 

```suggestion
- Create a custom alert rule.
- Create a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace. 
- Name the key, `custom_rules.yaml`. See the following example: 
< the example > 
- - *Note:*  You can create multiple rules in the configuration. 
```

Reviewer: It _seems_ to me that the "you can create multiple rules..." is a note, because it's an optional step for the user. If it's optional, you can say optional instead. Or you, you can simply make it another step, if you want the user to create multiple rules in the configuration

```suggestion
- Create a custom alert rule.
- Create a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace. 
- Name the key, `custom_rules.yaml`. See the following example: 
< the example > 
- - *Optional:*  You can create multiple rules in the configuration. 
```

Reviewer: ```suggestion
- Create a custom alert rule.
- Create a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace. 
- Name the key, `custom_rules.yaml`, and create rules in the configuration. See the following example: 
```

Reviewer: Pick whichever of these suggestions you think is best for the user experience :)

> - Create a custom alert rule. Create a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace. You must name the key, `custom_rules.yaml`, as shown in the following example. You can create multiple rules in the configuration. 

Author: Thanks Jake. The alert rule is created by creating the config map

```suggestion
- Create a custom alert rule by creating a config map named `observability-thanos-rule-custom-rules` in the `open-cluster-management-observability` namespace.

 ** Add the alert in the `custom_rules.yaml` section as shown in the following example.
```

            severity: critical
----
+
* The default alert rules are in the `observability-thanos-rule-default-rules` config map of the `open-cluster-management-observability` namespace.

> - Create a custom recording rule within the `observability-thanos-rule-custom-rules` config map. Create a recording rule that gives you the ability to get the sum of the container memory cache of a pod. Your YAML might resemble the following content:

Reviewer: Like my earlier comment, what do you think of making this into multiple steps? 

```suggestion
- Create a custom recording rule within the `observability-thanos-rule-custom-rules` config map.
-  Create a recording rule that gives you the ability to get the sum of the container memory cache of a pod. Your YAML might resemble the following content:
```
